### DISCUSSION TITLE
请问 Graph.fromJSON 有回调事件吗？

### DISCUSSION THREAD

**Discussion by songday** [antvis/X6#3752] [Q&A] [answered]
请问 Graph.fromJSON 有回调事件吗？
我想在fromJSON 结束之后执行一些方法

**Comment by NewByVector** [comment, marked as answer]
没有回调事件的，可以通过[视图事件](http://x6.antv.antgroup.com/tutorial/basic/events#%E8%A7%86%E5%9B%BE)来监听画布元素渲染完成。
